examples/stm32h750vbt6_devebox/blinky/openocd.cfg

```
@@ -0,0 +1 @@
+source [find interface/stlink-v2.cfg]
```

```suggestion
source [find interface/stlink.cfg]
```

With OpenOCD 0.11 (?) the `stlink-v2.cfg` file ist deprecated, as far as I remember.